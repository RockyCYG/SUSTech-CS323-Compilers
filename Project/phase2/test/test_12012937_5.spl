struct A{
	int apex;
	int my;
	float eyes;

};
int cpp()
{
	struct A Pheobe;
	struct Pheobe A;
	Pheobe.eyes=Pheobe.my;
}
