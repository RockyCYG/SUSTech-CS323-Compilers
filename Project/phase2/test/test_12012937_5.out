Error type 1 at Line 10: undefined variable: A
Error type 5 at Line 11: unmatching type on both sides of assignment
